ConeSearch.tex

```diff
@@ -16,58 +12,65 @@
 
 \ivoagroup{Data Access Layer}
 
-\author[http://www.ivoa.net/twiki/bin/view/IVOA/MarcoMolinaro]{Marco Molinaro}
-\author[http://www.ivoa.net/twiki/bin/view/IVOA/AdaNebot]{Ada Nebot}
-\author[http://www.ivoa.net/twiki/bin/view/IVOA/MarkusDemleitner]{Markus Demleitner}
-\author[http://www.ivoa.net/twiki/bin/view/IVOA/BobHanisch]{Robert Hanisch}
-\author[http://www.ivoa.net/twiki/bin/view/IVOA/RayPlante]{Raymond Plante}
-\author[http://www.ivoa.net/twiki/bin/view/IVOA/AlexSzalay]{Alex Szalay}
-\author[http://www.ivoa.net/twiki/bin/view/IVOA/RoyWilliams]{Roy Williams}
+\author[http://www.ivoa.net/twiki/bin/view/IVOA/MarcoMolinaro]{Marco
```

Comment:
I think the old layout of the author list was easier to read, despite varying in length

Reply:
fixed in commit 98611e2